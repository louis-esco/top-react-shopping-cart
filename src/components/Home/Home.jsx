import styles from "./Home.module.css";
import hand from "../../assets/hand.svg";

export default function Home() {
  return (
    <div className={styles.homeContainer}>
      <img src={hand} alt="Hand waving" />
      <h2 className={styles.welcomeMsg}>Welcome to Louis&apos;s shop ! </h2>
      <div>
        We sell all sorts of stuff, hopefully you&apos; find what you need here
        !
      </div>
    </div>
  );
}
