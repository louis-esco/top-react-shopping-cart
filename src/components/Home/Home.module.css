.homeContainer {
  min-height: calc(100vh - 150px);
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
}

.welcomeMsg {
  font-family: "Segoe Print", "Bradley Hand", Chilanka, TSCu_Comic, casual,
    cursive;
  font-size: 2.75rem;
}
