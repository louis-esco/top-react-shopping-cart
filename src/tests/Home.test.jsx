import { describe, it, expect } from "vitest";
import { render } from "@testing-library/react";
import Home from "../components/Home/Home";

describe("Home component", () => {
  it("Renders Home", () => {
    const { container } = render(<Home />);

    expect(container).toMatchSnapshot();
  });
});
