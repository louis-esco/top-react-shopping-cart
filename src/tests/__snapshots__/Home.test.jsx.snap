// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Home component > Renders Home 1`] = `
<div>
  <div
    class="_homeContainer_345077"
  >
    <img
      alt="Hand waving"
      src="/src/assets/hand.svg"
    />
    <h2
      class="_welcomeMsg_345077"
    >
      Welcome to Louis's shop ! 
    </h2>
    <div>
      We sell all sorts of stuff, hopefully you'll find what you need here !
    </div>
  </div>
</div>
`;

exports[`Navbar component > Renders Home 1`] = `
<div>
  <div
    class="_homeContainer_345077"
  >
    <img
      alt="Hand waving"
      src="/src/assets/hand.svg"
    />
    <h2
      class="_welcomeMsg_345077"
    >
      Welcome to Louis's shop ! 
    </h2>
    <div>
      We sell all sorts of stuff, hopefully you'll find what you need here !
    </div>
  </div>
</div>
`;
